Docs/Configuration.tex

```diff
@@ -1498,9 +1498,14 @@ \subsection{Quirks Properties}\label{booterpropsquirks}
   This option is relevant to users with issues booting to safe mode
   (e.g. by holding \texttt{shift} or with using the \texttt{-x} boot argument). By
   default, safe mode forces \texttt{0} slide as if the system was launched with the
-  \texttt{slide=0} boot argument. This quirk attempts to patch \texttt{boot.efi} to remove
-  this limitation and let some other value (from \texttt{1} to \texttt{255}) be used.
-  This quirk requires enabling \texttt{ProvideCustomSlide}.
+  \texttt{slide=0} boot argument.
+
+  \begin{itemize}
+    \tightlist
+    \item This quirk attempts to patch the \texttt{boot.efi} file to remove this limitation and
+      to allow using other values (\texttt{1} to \texttt{255} inclusive).
+    \item This quirk requires enabling \texttt{ProvideCustomSlide}.
```

Comment:
`This quirk requires \texttt{ProvideCustomSlide} to be enabled.` sounds better?

Reply:
These are instructions/specifications (telling users what to do). An active voice is needed.